app/assets/javascripts/views/postChatSurvey/PostChatSurveyDigitalAssistant.js

```diff
@@ -1,8 +1,6 @@
 const html = `
     <div id="postChatSurvey">
-        <legend class="govuk-fieldset__legend govuk-fieldset__legend--l" id="legend_give_feedback" tabindex="-1">
-          <h1 class="govuk-fieldset__heading">Give feedback</h1>
-        </legend>
+          <h2 class="govuk-fieldset__heading" id="legend_give_feedback" tabindex="-1">Give feedback</h2>
```

Comment:
Hi @pratikrana06 I think we can remove the class from this h2 as it's not within a fieldset. Can we remove it and see what it looks like?

Reply:
> Hi @pratikrana06 I think we can remove the class from this h2 as it's not within a fieldset. Can we remove it and see what it looks like?

Done